src/main/resources/assets/computercraft/lua/rom/programs/gps.lua

```diff
@@ -87,11 +87,6 @@ elseif sCommand == "host" then
             end
         end
     end
-
-    -- Close the channel
-    print( "Closing channel" )
```

Comment:
If you read the code you will see that those lines were never called. The while true end loop don't have any 'break' in it so this lines are never reached.